src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Cancellation.cs

```diff
@@ -123,25 +129,33 @@ public async Task GetAsync_CancelDuringResponseHeadersReceived_TaskCanceledQuick
         [MemberData(nameof(TwoBoolsAndCancellationMode))]
         public async Task GetAsync_CancelDuringResponseBodyReceived_Buffered_TaskCanceledQuickly(bool chunkedTransfer, bool connectionClose, CancellationMode mode)
         {
+            if (LoopbackServerFactory.IsHttp2 && (chunkedTransfer || connectionClose))
+            {
+                // There is no chunked encoding or connection header in HTTP/2
```

Comment:
Same question as above.

Reply:
This comment is right as Http2 does not really have "Connection: close" and connection management is independent of individual requests. 
It would be nice IMHO if we do not generate incompatible parameter set for HTTP2 but so far I did not find any good way how to do that. If anybody has recommendation, please let me know.